### DISCUSSION TITLE
Template reuse the {{VALUE:<name>}} in multiple locations

### DISCUSSION THREAD

**Discussion by ebmarquez** [chhoumann/quickadd#500] [General]
Is it possible to reuse the same value in multiple locations without having the type it for every instance it appears.

For example (template)
File name can be Personal/{{VALUE:State}}/{{VALUE:Address}} 

in Capture format section  include:
State::{{VALUE:State}}
Address::{VALUE:Address}}

I don't want to have to type the same entry again.

**Comment by chhoumann** [comment]
Hi @ebmarquez - values with the same name are reused. Is this not the case for you? What's your workflow like where this doesn't happen?

**Reply by ebmarquez** [reply]
I must have had the syntax case slightly different. I can see it working now.

**Reply by chhoumann** [reply]
Glad to hear it's working now
